examples/getting-started-with-serverless-framework/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "bolt-js-getting-started-with-serverless-framework-app",
+  "version": "1.0.0",
+  "description": "Getting Started with Serverless Framework ⚡️ Bolt for JavaScript",
+  "main": "app.js",
+  "scripts": {
+    "ngrok": "ngrok http 3000",
```

Comment:
I like that you added this a `npm script`, but I don't see you reference it in the guide itself. I guess one issue with it would be that once you run this command, it will take over that terminal window

Reply:
Thanks! This is an artifact from another sample that I made. 

Unfortunately, I've noticed that `npx ngrok` doesn't work (arguments in the `ngrok` script are not passed down to npx).

I'll remove it from the scripts because it's not adding very much value. In the guide, we need to explicitly mention `ngrok http 3000` because some developers may be following along with their own Bolt apps.  See commit f14a449